modules/learn/pages/clusters-and-availability/xdcr-viewing-conflict-logs.adoc

```diff
@@ -0,0 +1,348 @@
+
+You can access, view, and process these documents programmatically, similar to your application documents, using the available utilities, features, and options.
+
+For example, you can use RBAC to manage access to conflict collections.
+You can also use Couchbase services and tools such as Eventing and Kafka Connect to process conflict log documents.
+
+For information about the Conflict Record Document, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record].
+
+[#using-sdk-sqlpp-for-conflict-logs]
+== Using Couchbase SDK with SQL++ for Conflict Logs
+
+The most common way to view these documents is by using the Couchbase SDK with SQL++.
+
+The following are a few examples.
+In the examples, the conflict logs are stored in the collection `conflictlogs.bucket1.app1logs`.
+
+[#sqlpp-examples-clog]
+=== SQL++ Examples
+
+For queries, you can create a custom index or a primary index.
+
+* Create a primary index as follows.
++
+[source,sqlpp]
+----
+CREATE PRIMARY INDEX `#primary` ON `conflictlogs`.`bucket1`.`app1logs`
+----
+
+* Create a custom index as follows.
++
+You can use the `WHERE` clause with `SUBSTR(META().id,0,3) = “crd”` by making it an indexed field instead of `WHERE META().id LIKE "crd%"`.
++
+For example:
++
+[source,sqlpp]
+----
+CREATE INDEX `idx_crd_timestamp_docId` ON `conflictlogs`.`bucket1`.`applogs`(SUBSTR(META().id,0,3), `timestamp`,`docId`)
+----
++
+NOTE: When creating custom indexes, `META().id` is already a part of the index key.
+When using `LIKE` in the queries, underscore is a wildcard. 
+If you use underscore in comparison strings, ensure you use the escape character, so that it's used as an exact comparison instead of a wildcard comparison.
+For example, `WHERE META().id LIKE “crd\\_%”`.
+
+*Query example 1:* List all `docId` that had conflicts between two time periods.
+
+[source,sqlpp]
+----
+SELECT META().id, b.docId, b.timestamp
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META().id LIKE "crd%" and b.timestamp between "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+----
+
+*Query example 2:* Number of conflicts logged between two time periods.
+
+[source,sqlpp]
+----
+SELECT count(*) AS num_conflicts
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META().id LIKE "crd%" and b.timestamp between "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+----
+
+*Query example 3:* Number of different `docIds` that had conflicts between two time periods.
+
+[source,sqlpp]
+----
+SELECT count (distinct b.docId) as num_different_docIds
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META().id LIKE "crd%" and b.timestamp between "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+----
+
+*Query example 4:* List all `docIds` and the number of times each `docId` had a logged conflict between two time periods.
```

Comment:
(Add the below info.)

A query like this may be of interest if you are suspecting that same few documents are being updated multiple times concurrently in two different clusters, generating conflicts.